src/reactionsystem_conversions.jl

```diff
@@ -43,22 +48,33 @@ function oderatelaw(rx; combinatoric_ratelaw = true)
 end
 
 # Function returning `true` for species which shouldn't change from the reactions, 
-# including non-species variables.
-drop_dynamics(s) = isconstant(s) || isbc(s) || (!isspecies(s))
-
-function assemble_oderhs(rs, ispcs; combinatoric_ratelaws = true, remove_conserved = false)
+# including non-species variables
+drop_dynamics(spec) = isconstant(spec) || isbc(spec) || (!isspecies(spec))
+
+# for a `ReactionSystem`, create a vector with all the right-hand side expressions in the
+# corresponding ODE system (used by the `assemble_drift` function)
+function assemble_oderhs(rs, isps; combinatoric_ratelaws = true, remove_conserved = false)
```

Comment:
Again, `ispcs` to `isps` (previously we used s, sp, spc, and spec to denote species, now we use sp or spec, which is a little bit less versions at least) if you want I can change these back